ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlServerMain.java

```diff
@@ -48,8 +50,8 @@ public static void main(final String[] args) {
       );
 
       final String installDir = properties.getOrDefault("ksql.server.install.dir", "");
-      final String streamsStateDirPath = properties.getOrDefault("ksql.streams.state.dir",
-          getSampleStreamsConfig().getString(StreamsConfig.STATE_DIR_CONFIG));
+      final String streamsStateDirPath = StreamsConfig.configDef().defaultValues()
```

Comment:
How come we're always using the default value now? We still need to check `properties` in case the user has overridden the default.

Reply:
Ahh, my bad! Fixed it.